iphone/metadata/en-US/release_notes.txt

```diff
@@ -1,13 +1,24 @@
-• New OpenStreetMap data as of February 27
-• Fixed freeze on iPad when a keyboard is connected
-• Fixed unnecessary double click on the route button
-• Removed duplicated floors and zip code cells
-• Enabled OpenStreetMap editor notes for all selected places
-• Fixed and improved search
-• Split vegan and vegetarian cuisines in search
-• Fixed styles for airports, landuses, waterways, and hatching fill areas
-• Added healthcare specialists: alternative, audiologist, blood_donation, optometrist, podiatrist, psychotherapist, sample_collection, speech_therapist
-• Added communication towers and solar, wind, gas, and hydropower generators
-• Improved Arabic, Norwegian, Portuguese, and Turkish translations
+• New OSM Planet data as of March 26
+• Add localized links for the OSM wiki page
+• Replaced uphill and downhill icons with more clear arrows
+• Correctly import KML bookmarks with track data
+• Disable 3D buildings preference if power saving is enabled.
+• [editor] Allow adding music and musical_instruments.
+• Sewing shops and haberdasheries are searchable
+• Distinguish supermarkets and convenience stores
+
+Styles
+• New shop types: agrarian, appliance, bed, outpost, gas, key cutter, locksmith, carpet, motorcycle repair, pet grooming, hi-fi, bathroom furnishing, curtain, fashion accessories, cannabis, herbalist
+• Added money transfer, bicycle charging stations, power plant
+• Made captions mandatory for capitals with a star icon
+• Made park, forest, and scrub colors different
+• Added metal construction icons
+• Improved aerial ways, cable cars, power lines, fences
+• Reduce Barcelona subway visibility
+• Fixed isolines that are drawn over buildings, roads, etc.
+
```

Comment:
* Fixed rendering of multiple-type features (e.g. some POI captions were displayed too early)

(https://github.com/organicmaps/organicmaps/pull/4835)